app/views/task_managements/index.html.erb

```diff
@@ -1,8 +1,35 @@
 <main>
-  <section class="col m9 s12 offset-m1">
-    <div class="row">
-      <ul class= "col m10 offset-m1" >
-        <h2>My Tasks</h2>
+  <%= render "partials/sidebar_dashboard" %>
+  <div class="row">
+    <section class="col m9 offset-m2">
+      <h2>Task logs</h2>
+      <% if current_user.user_type == "taskee" %>
+          <table>
+              <thead>
+                  <tr>
+                      <td>Tasker</td>
+                      <td>Description</td>
+                      <td>Status</td>
+                  </tr>
+              </thead>
+              <tbody>
+                  <% @tasks.each do |task| %>
+                      <tr>
+                          <% user = User.find(task.tasker_id) %>
+                          <td><%= "#{user.firstname} #{user.lastname}" %></td>
+                          <td><%= task.task_desc %></td>
+                          <% if task.status.downcase == "done" %>
```

Comment:
This whole logic can be extracted into a helper method or a decorator method. Strive for cleaner html pages, pushing logic into a helper, a decorator or a view object/presenter.